.env.example

```
@@ -45,5 +45,5 @@ PUBLIC_ORG_ID="dgen-dev"
 # Configure your widget at https://swapspace.co/affiliate/widget
 PUBLIC_SWAPSPACE_WIDGET_URL=https://swapspace.co/widget/b04b927eb4e70d06b09928eb
 
-# DGEN Credit Card is live Url
-PUBLIC_CREDITCARD_IS_LIVE_URL=https://card.dgentech.io/pages/r?c=AMeaqiij
\ No newline at end of file
+# DGEN Card is live Url
+PUBLIC_DGENCARD_IS_LIVE_URL=https://card.dgentech.io/pages/r?c=AMeaqiij
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)